homeassistant/components/shelly/coordinator.py

```diff
@@ -661,6 +662,7 @@ def _async_handle_update(
         self, device_: RpcDevice, update_type: RpcUpdateType
     ) -> None:
         """Handle device update."""
+        LOGGER.debug("Shelly %s handle update, type: %s", self.name, update_type)
```

Comment:
This was added while I was debugging the change and I noticed it may be useful since it is the only way to know which type of update we are processing so I left it and added to block device also.